docs/authentication/configuration/sign-up-sign-in-options.mdx

```diff
@@ -187,14 +191,17 @@ Clerk currently offers the following MFA strategies:
 - **Authenticator application (also known as TOTP - Time-based One-time Password)**
 - **Backup codes**
 
-Enabling MFA allows users of your app to turn on MFA for their own account through their [User Profile](/docs/customization/account-portal//user-profile-org-profile#user-profile) page. Enabling MFA does not automatically turn on MFA for all users.
+Enabling MFA allows users of your app to turn it on for their own accounts through their [User Profile](/docs/customization/account-portal//user-profile-org-profile#user-profile) page. Enabling MFA does not automatically activate MFA for all users.
```

Comment:
The user profile can be accessed through the Account Portal user profile page, or the `<UserProfile />` component, or a custom flow. So let's just say "user profile page" here.

```suggestion
Enabling MFA in the Clerk Dashboard allows users of your app to turn it on for their own accounts through their user profile; it does not automatically activate MFA for all users.
```